protocol/wlr-screencopy-unstable-v1.xml

```diff
@@ -102,6 +102,12 @@
 
         The client should then create a buffer with the provided attributes, and
         send a "copy" request.
+
+        When the buffer is a dmabuf, the stride parameter may be ignored.
```

Comment:
These changes need to be sent to wlr-protocols

Reply:
https://github.com/swaywm/wlr-protocols/pull/80